content/blog/command-provider-ga/index.md

```diff
@@ -0,0 +1,221 @@
+
+        var petFileCommand = new Command("petFileCommand", CommandArgs.builder()
+            .create(pet.id().applyValue(id -> String.format("touch \"%s.txt\"", id)))
+            .delete(pet.id().applyValue(id -> String.format("rm \"%s.txt\"", id)))
+            .triggers(pet.id())
+            .build());
+    }
+}
+```
+
+{% /choosable %}}
+
+{{% choosable language typescript %}}
+
+```typescript
+import * as pulumi from "@pulumi/pulumi";
+import * as command from "@pulumi/command";
+import * as random from "@pulumi/random";
+
+const pet = new random.RandomPet("pet", {});
+
+const petFileCommand = new command.local.Command("petFileCommand", {
+    create: pulumi.interpolate`touch "${pet.id}.txt"`,
+    delete: pulumi.interpolate`rm "${pet.id}.txt"`,
+    // Replace the resource, i.e., re-run the 'create' command,
+    // when the pet resource changes.
+    triggers: [pet.id],
+});
+```
+
+{{% /choosable %}}
+
+{{% choosable language python %}}
+
+```python
+import pulumi
+import pulumi_command as command
+import pulumi_random as random
+
+pet = random.RandomPet("pet")
+
+pet_file_command = command.local.Command("petFileCommand",
+    create=pet.id.apply(lambda id: f"touch \"{id}.txt\""),
+    delete=pet.id.apply(lambda id: f"rm \"{id}.txt\""),
+    # Replace the resource, i.e., re-run the 'create' command,
+    # when the pet resource changes.
+    triggers=[pet.id])
+```
+
+{{% /choosable %}}
+
+{{% choosable language go %}}
+
+```go
+package main
+
+import (
+	"fmt"
+
+	"github.com/pulumi/pulumi-command/sdk/go/command/local"
+	"github.com/pulumi/pulumi-random/sdk/v4/go/random"
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
+)
+
+func main() {
+	pulumi.Run(func(ctx *pulumi.Context) error {
+		pet, err := random.NewRandomPet(ctx, "pet", nil)
+		if err != nil {
+			return err
+		}
+		_, err = local.NewCommand(ctx, "petFileCommand", &local.CommandArgs{
+			Create: pulumi.String(fmt.Sprintf("touch \"%v.txt\"", pet.ID())),
+			Delete: pulumi.String(fmt.Sprintf("rm \"%v.txt\"", pet.ID())),
+			// Replace the resource, i.e., re-run the 'create' command,
+			// when the pet resource changes.
+			Triggers: pulumi.Array{ pet.ID() },
+		})
+		if err != nil {
+			return err
+		}
+		return nil
+	})
+}
+```
+
+{{% /choosable %}}
+
+{{% choosable language csharp %}}
+
+```csharp
+using Pulumi;
+using Command = Pulumi.Command;
+using Random = Pulumi.Random;
+
+return await Deployment.RunAsync(() =>
+{
+    var pet = new Random.RandomPet("pet");
+
+    var petFileCommand = new Command.Local.Command("petFileCommand", new()
+    {
+        Create = pet.Id.Apply(id => $"touch \"{id}.txt\""),
+        Delete = pet.Id.Apply(id => $"rm \"{id}.txt\""),
+        Triggers = new[]
+        {
+            pet.Id,
+        },
+    });
+});
+```
+
+{{% /choosable %}}
+
+{{< /chooser >}}
+
+Rather than simply putting a “v1” label on the existing provider, we used this opportunity to enhance it with some previously requested capabilities.
```

Comment:
I think we need to say more about what 1.0 means:
1. A stable API for the 1.x series
2. A complete set of capabilities for the core user needs for the Command provider

And then talk about a few of the recent things we've done to finalize (2) over the last few months ahead of the 1.0 release.